docs(notices): add Upgrade 19 activation dates for Sepolia and Mainnet (#21039)
ethereum-optimism/optimism: docs/public-docs/notices/upgrade-19.mdx

</Warning>

<Info>

>   The Upgrade 19 protocol upgrade on **Sepolia OP Stack chains** will be executed on **Wed, Jun 17, 2026 at 16:00:01 UTC** (Unix `1781712001`). On **Mainnet OP Stack chains** the upgrade is planned to optimistically activate on **Wed, Jul 8, 2026 at 16:00:01 UTC** (Unix `1783526401`), depending on Optimism Governance's approval.

Reviewer: ```suggestion
  The Upgrade 19 protocol upgrade on Optimism Governed **Sepolia OP Stack chains** will be activate on **Wed, Jun 17, 2026 at 16:00:01 UTC** (Unix `1781712001`). On **Mainnet OP Stack chains** the upgrade is planned to optimistically activate on **Wed, Jul 8, 2026 at 16:00:01 UTC** (Unix `1783526401`), depending on Optimism Governance's approval.
```

>   The Upgrade 19 protocol upgrade on Optimism Governed **Sepolia OP Stack chains** will be activate on **Wed, Jun 17, 2026 at 16:00:01 UTC** (Unix `1781712001`). On **Mainnet OP Stack chains** the upgrade is planned to optimistically activate on **Wed, Jul 8, 2026 at 16:00:01 UTC** (Unix `1783526401`), depending on Optimism Governance's approval.

Reviewer: ```suggestion
  The Upgrade 19 protocol upgrade on Optimism Governed **Sepolia OP Stack chains** will be activated on **Wed, Jun 17, 2026 at 16:00:01 UTC** (`1781712001`). On **Mainnet OP Stack chains** the upgrade is planned to optimistically activate on **Wed, Jul 8, 2026 at 16:00:01 UTC** (`1783526401`), depending on Optimism Governance's approval. The smart contract upgrades are expected to happen the week prior to activation. 
```

  The upgrade will be executed on the following chains: `OP`, `Soneium`, `Ink`, `Unichain`, `Mode`, `Metal`, and `Zora`.

>   Upgrade 19 includes hard fork changes (Osaka on L2, Glamsterdam Defense and hard fork activated L2 Contract upgrades). Node operators must upgrade before the activation timestamp.

Reviewer: ```suggestion
```

Reviewer: I think we can remove thjis because the rest of the page goes into these details
